packages/nvidia_nat_langchain/src/nat/plugins/langchain/langgraph_workflow.py

```
@@ -125,13 +124,24 @@ async def _astream(self, value: LanggraphWrapperInput) -> AsyncGenerator[Langgra
                 logger.info("Graph is an async context manager")
                 async with self._graph as graph:
                     async for output in graph.astream(value.model_dump()):
-                        yield LanggraphWrapperOutput.model_validate(output)
+                        yield self._parse_stream_output(output)
             else:
                 async for output in self._graph.astream(value.model_dump()):
-                    yield LanggraphWrapperOutput.model_validate(output)
+                    yield self._parse_stream_output(output)
         except Exception as e:
             raise RuntimeError(f"Error in LangGraph workflow: {e}") from e
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)